test/index.js

```diff
@@ -578,27 +580,215 @@ describe('views()', () => {
 
 describe('Plugin', () => {
 
-    it('only registers once', async () => {
+    it('registers multiple times', async () => {
 
         const one = {
             name: 'one',
-
             register: function (server, options) {
 
-                return server.register(Vision);
+                server.register({
```

Comment:
Should we be using awaiting the return of this? (seen later in other call sites as well)

```js
await server.register(...
```

Reply:
I've asked around about this, "what's the difference between `return await [Promise]` and `return [Promise]`? And `ljharb` in the `Node.js` Slack community said they're identical in what they do, only returning an await causes an unnecessary wrapper. Do you know if there's a difference?

Reply:
I see. That's good to know.

> Do you know if there's a difference?

Nope. I'm new to async/await in general.

Reply:
I find it's generally easier to remove cognitive load by simply awaiting anything async– only think about the underlying promises when really necessary.  For what it's worth, in a lot of hapi code you'll find `return await somePromise`.  Unless perf is terribly important, that seems most consistent.

Reply:
Good to know!!